images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -37,6 +37,21 @@ function Get-RustVersion {
     return "Rust ${rustVersion}"
 }
 
+function Get-BindgenVersion {
+    return & bindgen --version
```

Comment:
Why do we need &?

Reply:
I think we don't need, just to be consistent with other checks in file. Would it be better to remove `&` ?

Reply:
Removed &